infra/aws/terraform/prow-build-cluster/resources/monitoring/prometheus-main/sa.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: monitoring
+  # annotations:
+  #   eks.amazonaws.com/role-arn: "arn:aws:iam::<acc-id>:role/prometheus-demo"
```

Comment:
Can you leave some comment why is this commented? I assume this was when we used managed Prometheus offering. If that's the case, we can probably remove this.

Reply:
You are right. That's a leftover.